gremlin-dotnet/src/Gremlin.Net/Process/Utils.cs

```diff
@@ -63,5 +64,23 @@ public static void Forget(this Task task)
                 t.Exception?.Handle(_ => true);
             }, TaskContinuationOptions.ExecuteSynchronously);
         }
+
+        /// <summary>
+        ///  Returns a user agent for connection request headers.
+        ///
+        /// Format:
+        /// "[Application Name] [GLV Name].[Version] [Language Runtime Version] [OS].[Version] [CPU Architecture]"
+        /// </summary>
+        public static string GetUserAgent()
+        {
+            string applicationName = Assembly.GetExecutingAssembly().GetName().Name.Replace(' ', '_');
```

Comment:
project convention is to use `var` keyword for declarations

```suggestion
            var applicationName = Assembly.GetExecutingAssembly().GetName().Name.Replace(' ', '_');
```